config/grafana/dashboards/resources-metrics.json

```diff
@@ -0,0 +1,349 @@
+{
```

Comment:
The file you removed was generated by this `make grafana-dashboard` command which is configured in the makefile. They are automatically generated by a kubebuilder plugin `grafana.kubebuilder.io/v1-alpha`

Reply:
As discussed I deleted the make target and the kubebuilder plugin generated dashboards. They don't work and we keep our manually created dashboard only.